.github/workflows/deploy-i18n.yml

```diff
@@ -134,10 +134,54 @@ jobs:
           docker push registry.digitalocean.com/${{ secrets.DOCR_NAME }}/${{ matrix.site_tlds }}/news-${{ matrix.languages }}:$tagname
           docker push registry.digitalocean.com/${{ secrets.DOCR_NAME }}/${{ matrix.site_tlds }}/news-${{ matrix.languages }}:latest
 
+      - name: Clean old DO images
+        uses: actions/github-script
```

Comment:
I would have written this in bash, but parsing the JSON into a bash array was becoming an absolute nightmare.

Reply:
Have you used `jq`? It's going to change the way you look at json forever. https://jqplay.org

Reply:
Life-saver. I've pushed a bash version.